tests/migration/migration.go

```diff
@@ -752,6 +754,58 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 			})
 		})
 
+		Context("[Serial] already running", Serial, func() {
+			var vmi *v1.VirtualMachineInstance
+
+			BeforeEach(func() {
+				tests.EnableDownwardMetrics(virtClient)
+
+				vmi = libvmifact.NewFedora(
+					libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+					libvmi.WithNetwork(v1.DefaultPodNetwork()),
+					libvmi.WithDownwardMetricsChannel(),
+				)
+				vmi = libvmops.RunVMIAndExpectLaunch(vmi, 180)
+				Expect(console.LoginToFedora(vmi)).To(Succeed())
+
+				tests.DisableDownwardMetrics(virtClient)
+
+			})
+
+			AfterEach(func() {
+				tests.EnableDownwardMetrics(virtClient)
```

Comment:
Should be reset to original value, i.e. reset the KV object after? I think we have helpers for that.

Reply:
What would be the benefit of reset the entire KV object? IMHO, if we are only changing one specific value, it makes sense to just reset that value back.